site/src/sphinx/setup-configuration.rst

```diff
@@ -77,6 +79,23 @@ Core properties
   - the configuration for Transport Layer Security(TLS) support. Specify ``null`` to disable TLS.
     See :ref:`tls` for more information.
 
+- ``trustedProxyAddresses`` (string array)
+
+  - the addresses or ranges of `Classless Inter-domain Routing (CIDR) <https://tools.ietf.org/html/rfc4632>`_
+    blocks of trusted proxy servers. e.g. ``10.0.0.1`` for a single address or ``10.0.0.0/8`` for a CIDR block.
+    With ``trustedProxyAddresses`` and ``clientAddressSources`` properties, you can get a client address
+    who initiated a request from the access log. If ``null`` or an empty array, a remote endpoint of
```

Comment:
a remote endpoint of the connected socket -> the remote address of the connection?